Packs/DataBee/Integrations/DataBee/DataBee.py

```diff
@@ -0,0 +1,839 @@
+    additional_context: list[AdditionalContext],
+) -> CommandResults:
+    """
+    Search for DataBee tables.
+
+    Args:
+        client (Client): DataBee API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    limit, offset = get_pagination_args(
+        page=args["page"],
+        limit=args["limit"],
+        page_size=args.get("page_size"),
+    )
+
+    response = client.search(
+        table=settings.type,
+        query=build_full_query(search_type=settings.type, args=args),
+        limit=limit,
+        offset=offset,
+    ).json()
+    return generate_command_results(
+        title=f"{settings.title} List",
+        outputs_prefix=settings.output_prefix,
+        outputs_key_field="uid",
+        headers=settings.headers,
+        outputs=parse_response(
+            type=settings.type,
+            data=response["results"],
+            keys=settings.output_keys,
+            additional_context=additional_context,
+        ),
+        raw_response=response,
+    )
+
+
+def get_endpoint_command(
+    client: Client,
+    args: dict[str, Any],
+) -> list[CommandResults]:
+    hostname = args.get("hostname")
+    ip = args.get("ip")
+    id = args.get("id")
+
+    if not ip and not hostname and not id:
+        # in order not to return all the devices
+        raise ValueError("Please add a filter argument - ip, hostname ot id.")
+
+    # use OR operator between filters (https://github.com/demisto/etc/issues/46353)
+    raw_res = client.search(
+        table=SEARCH_CONFIGURATIONS[SearchTypes.DEVICE].type,
+        query=build_full_query(
+            search_type=SEARCH_CONFIGURATIONS[SearchTypes.DEVICE].type,
+            args=(args | {"search_operator": "in"}),
+        ),
+    ).json()
+
+    devices = raw_res.get("results", [])
+    if not devices:
+        raise ValueError("No devices was found")
+
+    standard_endpoints = []
+    for single_device in devices:
+        endpoint = Common.Endpoint(
+            id=single_device.get("uid"),
+            hostname=single_device.get("hostname"),
+            ip_address=single_device.get("ip"),
+            os=dict_safe_get(single_device, ["os", "type"]),
+            os_version=dict_safe_get(single_device, ["os", "version"]),
+            mac_address=single_device.get("mac"),
+            vendor=INTEGRATION_PREFIX,
+        )
+        standard_endpoints.append(endpoint)
+
+    command_results = []
+    for endpoint in standard_endpoints:
+        endpoint_context = endpoint.to_context().get(Common.Endpoint.CONTEXT_PATH)
+        hr = tableToMarkdown("DataBee Endpoint", endpoint_context)
+
+        command_results.append(
+            CommandResults(readable_output=hr, raw_response=raw_res, indicator=endpoint)
+        )
+    return command_results
+
+
+def normalize_finding(data: dict[str, Any], additional_context: list[AdditionalContext]):
```

Comment:
Consider moving this method to helper methods section 
Currently it is in the commands section